cli/ts/commands/poll.ts

```diff
@@ -55,7 +55,7 @@ export const getPoll = async ({
         `Deploy time: ${new Date(Number(deployTime) * 1000).toString()}`,
         `End time: ${new Date(Number(deployTime + duration) * 1000).toString()}`,
         `Number of signups ${numSignups}`,
-        `State Aq merged: ${isStateAqMerged}`,
+        `State Aq merged: ${mergedStateRoot}`,
```

Comment:
could you please change the text to `State tree merged`? to indicate is not an Aq anymore